### PR TITLE
en.chat4o.com

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#91 #90 
#### What does this implement/fix? Explain your changes.
- 27 new 1-line art added
        1. mouse4
	2. pistols3
	3. rock on2
	4. love in my eye2
	5. love in my eye3
	6. sad3
	7. sad4
	8. headphone2
	9. cigarette3
	10. spider3
	11. barcode2
	12. equalizer
	13. boombox2
	14. surprised2
        15. playing cards
        16. playing cards hearts
	17. playing cards diamonds
	18. playing cards spades
	19. playing cards clubs
	20. spade bold
	21. heart bold
	22. diamond bold
	23. club bold
	24. spade regular
	25. heart regular
	26. diamond regular
	27. club regular
- `barcode` 1-line art renamed to `barcode1`
- `cigaret` 1-line art renamed to `cigarette1`
- `cigarette` 1-line art renamed to `cigarette2`
- `headphone` 1-line art renamed to `headphone1`
- `rock on` 1-line art renamed to `rock on1`
- `boom box` 1-line art renamed to `boombox1`
- `love in my eye` 1-line art renamed to `love in my eye1`

#### Any other comments?
After this PR, we will be good to go for next release (`version 4.4`)

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
I some new minor edits in `README.md`, `INSTALL.md` and `travis.yml`

File: art/art_dic.py [hunk 1]
```diff
@@ -272,7 +279,9 @@
     "facepalm": "(>ლ)",
     "yo": "__o000o__(o)(o)__o000o__",
     "badass": "(⌐■_■)--︻╦╤─ - - -",
-    "cigaret": "()___)____________)",
+    "cigarette1": "()___)____________)",
+    "cigarette2": "(____((____________()~~~",
+    "cigarette3": "(̅_̅_̅_̅(̅_̅_̅_̅_̅_̅_̅_̅_̅̅_̅()ڪے",
```
Comment (sadrasabouri) [inline art/art_dic.py]: could you please change `cigarette1` and `cigarette3`?
`cigarette3` seems to be a more realistic model for cigarette :smoking:

File: art/art_dic.py [hunk 2]
```diff
@@ -468,5 +468,6 @@
     "surprised2": "(O_o)",
     "boombox2": "♫♪ |̲̅̅●̲̅̅|̲̅̅=̲̅̅|̲̅̅●̲̅̅| ♫♪",
     "equalizer": "▇ ▅ █ ▅ ▇ ▂ ▃ ▁ ▁ ▅ ▃ ▅ ▅ ▄ ▅ ▇",
+    "playing cards": "[♥]]] [♦]]] [♣]]] [♠]]]"
```
Comment (sadrasabouri) [inline art/art_dic.py]: can we divide this art into 4 art?
like `"playing cards hearts" : "[♥]]]"`

File: art/art_dic.py (hunk 1 above)
Comment (sepandhaghighi) [reply art/art_dic.py]: Good point!
Done https://github.com/sepandhaghighi/art/pull/92/commits/750a601862b9da3bb14114027c5d0e6ae8f5e3a9

File: art/art_dic.py (hunk 2 above)
Comment (sepandhaghighi) [reply art/art_dic.py]: Good point ;-)
I added 12 new arts for individual mode https://github.com/sepandhaghighi/art/pull/92/commits/90e5ab4664efa8d8b242b6a4748490e80e0b17f4 https://github.com/sepandhaghighi/art/pull/92/commits/8bb2a4c1266f6aaeb4963118155de3b8e670db9b
I also found [this](https://en.wikipedia.org/wiki/Playing_cards_in_Unicode) set, but I think  these playing cards are too small !!

Comment (sadrasabouri) [reply art/art_dic.py]: I checked the link, i think these are interesting sets and worth adding, i didn't catch your point about smallness of these sets.
if it is OK for i can add this link to [Font and Art suggestion](https://github.com/sepandhaghighi/art/issues/59)

File: art/art_dic.py [hunk 3]
```diff
@@ -472,6 +472,14 @@
...
-    "playing cards clubs": "[♣]]]"
+    "playing cards clubs": "[♣]]]",
+    "black spade": "♠",
+    "black heart": "♥",
+    "black diamond": "♦",
+    "black club": "♣",
+    "white spade": "♤",
+    "white heart": "♡",
+    "white diamond": "♢",
+    "white club": "♧"
```
Comment (sadrasabouri) [inline art/art_dic.py]: Can we consider a better name for these sets?
black sets are colorful in some operating systems and i think `bold heart` and so on and `regular hear` and ... or any other suitable name.

File: art/art_dic.py (hunk 2 above)
Comment (sepandhaghighi) [reply art/art_dic.py]: > I checked the link, i think these are interesting sets and worth adding, i didn't catch your point about smallness of these sets.
> if it is OK for i can add this link to [Font and Art suggestion](https://github.com/sepandhaghighi/art/issues/59)

Okay, It's up to you ;-)

File: art/art_dic.py (hunk 3 above)
Comment (sepandhaghighi) [reply art/art_dic.py]: Done! https://github.com/sepandhaghighi/art/pull/92/commits/7602192355dd1ad43d7b32daa72b9fed757733b3

**Pull request merged**